docs/README.md

```diff
@@ -20,19 +20,38 @@ The LogDNA agent is a blazingly fast, resource-efficient log collection client,
   * [Running as Non-Root](#running-as-non-root)
   * [Additional Installation Options](#additional-installation-options)
 * [Building](#building)
-  * [Building on Linux](#building-on-linux)
-  * [Building on Docker](#building-on-docker)
+  * [Building Docker image](#building-on-docker)
 * [Configuration](#configuration)
   * [Options](#options)
   * [Configuring the Environment](#configuring-the-environment)
   * [Configuring Lookback](#configuring-lookback)
 
 ## Managing Deployments
 
-The agent is supported for Kubernetes 1.9+ and OpenShift 4.6+ environments. 
+The agent is supported for Kubernetes<sup>:registered:</sup> 1.9+ and Red Hat<sup>:registered:</sup> OpenShift<sup>:registered:</sup> 4.5+ environments.
 
 ### Installing
 
+__NOTE__: The Kubernetes manifest YAML files in this repository (and referenced in this
+documentation) describe the version of the LogDNA Agent in the current commit
+tree and no other version of the LogDNA agent. If you apply the kubernetes manifest
+found in the current tree then your cluster _will_ be running the version described
+by the current commit which may not be ready for general use.
+
+You MUST ensure that the current tree is checked out to the branch/tag is the
+version you intend to install, if you install a pre-release version of the
+agent your logs may not be collected.
+
+To install a specific version you should ensure you checkout the tag for that
+version before applying any YAML files from the current tree.
+
+For example to install the a particular 2.2 beta you would run
+```bash
+git checkout 2.2.0-beta.10
+```
+in the repo's root directory before following the install instructions relevant
```

Comment:
NIT: code blocks with 3 backticks are represented as a `pre` block in html breaking the current `p` paragraph, so ideally it should be preceded and followed by paragraphs.

Also, some IDEs and markdown viewers (not the case for GH) don't like code blocks not separated by empty lines.